templates/etc/sysctl.d/routing.conf

```diff
@@ -0,0 +1,23 @@
+net.ipv4.conf.default.rp_filter=1
+net.ipv4.conf.all.rp_filter=0
+
+net.ipv4.ip_forward=1
+net.ipv6.conf.all.forwarding=1
+
+# Answer ICMP errors on the inbound connection interface
+net.ipv4.icmp_errors_use_inbound_ifaddr=1
+
+# Decrease tcp timeout
+net.ipv4.netfilter.ip_conntrack_tcp_timeout_established=<%=conntrack_udp_timeout%>
+net.ipv4.netfilter.ip_conntrack_udp_timeout=<%=conntrack_udp_timeout%>
+
+# Increase neighbor cache
+net.ipv4.neigh.default.gc_thresh1=2048
+net.ipv4.neigh.default.gc_thresh2=4096
+net.ipv4.neigh.default.gc_thresh3=8192
+net.ipv6.neigh.default.gc_thresh1=2048
+net.ipv6.neigh.default.gc_thresh2=4096
+net.ipv6.neigh.default.gc_thresh3=8192
+
+net.core.wmem_max=8388608
+net.core.wmem_default=8388608
```

Comment:
we just went back to the default value in hamburg.
such a buffer size would introduce a delay of nearly 700ms worst case when the gateway has a 100Mbit/s connection. we think that having small buffers will lead to a more usable freifunk in the end, because throughput intensive applications will not slow down everything else as well.